.doc_gen/metadata/glue_metadata.yaml

```diff
@@ -349,6 +448,20 @@ glue_Scenario_GetStartedCrawlersJobs:
     title: "Tutorial: Getting started with &GLU; Studio"
     url: glue/latest/ug/tutorial-create-job.html
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/Glue
+          sdkguide:
+          excerpts:
+            - description: Create a class that wraps &GLU; functions used that
+                are used in in the scenario.
```

Comment:
Create a class that wraps &GLU; functions that are used in the scenario.

Reply:
fixed